x-pack/plugins/security_solution/server/fleet_integration/handlers/create_default_policy.ts

```diff
@@ -154,5 +170,13 @@ const getCloudPolicyConfig = (policy: PolicyConfig): PolicyConfig => {
         session_data: true,
       },
     },
+    windows: {
+      ...policyWithDisabledProtections.windows,
+      antivirus_registration: {
+        ...policyWithDisabledProtections.windows.antivirus_registration,
+        enabled: false,
```

Comment:
i guess `enabled: false` should be removed here, setting it to `false` is the responsibility of `disableProtections()` above